src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -189,43 +190,50 @@ class ArchiveExplorer extends React.Component {
                 <div className="bp-ArchiveExplorer" data-resin-feature="archive">
                     <SearchBar onSearch={this.handleSearch} searchQuery={searchQuery} />
                     <Breadcrumbs fullPath={fullPath} onClick={this.handleBreadcrumbClick} view={view} />
-                    <VirtualizedTable
-                        rowData={itemList}
-                        rowGetter={this.getRowData(itemList)}
-                        sort={this.handleSort}
-                        sortBy={sortBy}
-                        sortDirection={sortDirection}
-                    >
-                        {intl => [
-                            <Column
-                                key={KEY_NAME}
-                                cellRenderer={itemNameCellRenderer(intl, this.handleItemClick)}
-                                dataKey={KEY_NAME}
-                                flexGrow={3}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('filename')}
-                                width={1}
-                            />,
-                            <Column
-                                key={KEY_MODIFIED_AT}
-                                cellRenderer={readableTimeCellRenderer}
-                                dataKey={KEY_MODIFIED_AT}
-                                flexGrow={2}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('last_modified_date')}
-                                width={1}
-                            />,
-                            <Column
-                                key={KEY_SIZE}
-                                cellRenderer={sizeCellRenderer()}
-                                dataKey={KEY_SIZE}
-                                flexGrow={1}
-                                headerRenderer={sortableColumnHeaderRenderer}
-                                label={__('size')}
-                                width={1}
-                            />,
-                        ]}
-                    </VirtualizedTable>
+                    <div className="bp-AutoSizerContainer">
```

Comment:
This isn't a new component, so it should probably be called `bp-ArchiveExplorer-table` or something.